e2e_test/batch/catalog/pg_settings.slt.part

```diff
@@ -23,6 +23,7 @@ user client_encoding
 user client_min_messages
 user create_compaction_group_for_mv
 user datestyle
+user enable_streaming_rate_limit
```

Comment:
The PR's idea looks good to me except the new session variable. How about simply using `streaming_rate_limit=-1` as a placeholder of being unlimited?

Reply:
We can also go with `unlimited`

Reply:
I like `streaming_rate_limit=-1`. Changing it.